cpp/src/barretenberg/serialize/msgpack_impl/schema_impl.hpp

```diff
@@ -48,7 +53,46 @@ struct MsgpackSchemaPacker : msgpack::packer<msgpack::sbuffer> {
      * @tparam T the object's type.
      * @param obj the object.
      */
-    template <typename T> void pack_schema(const T& obj) { msgpack_schema_pack(*this, obj); }
+    template <typename T> void pack_schema(const T& obj) { msgpack_schema_pack_wrapper(*this, obj); }
+
+    // Recurse over any templated containers
+    // Outputs e.g. ['vector', ['sub-type']]
+    template <typename... Args> void pack_template_type(const std::string& schema_name)
```

Comment:
Q from ignorance: Im not used to this pack syntax, is Args passed along outside of the parameters?

Reply:
It is a type-only parameter

Reply:
Normally this wouldn't be useful, but as this is just printing schemas its all we need